packages/spacecat-shared-ahrefs-client/src/index.js

```diff
@@ -174,4 +174,28 @@ export default class AhrefsAPIClient {
 
     return this.sendRequest('/site-explorer/metrics-history', queryParams);
   }
+
+  async getOrganicKeywords(url, country = 'us', keywordFilter = [], limit = 200) {
```

Comment:
missing input parameter validation

Reply:
Do we need a country, what if this is a global site?
I wouldn't set the default to be `us`, but rather none.

Reply:
yes the country is mandatory for this Ahrefs request because keywords are country specific.
For the Google API, this is not the case, there you can get them across all countries, which in some cases also creates confusing results (in this case needs to be filtered by subdomain or path, which results ultimately in the same issue).

I see two options for this:

- we extract the country from either url or hreflang tag
- we collect the keywords from the top-pages (already suggested by @iuliag through other channels). This might have some disadvantages as top-pages differ from keyword rankings. On the other hand, we can save an additional Ahrefs API request. We could also increase the limit in this case to import maybe top 2k pages for better accuracy